packages/contractkit/src/identity/claims/claim.ts

```diff
@@ -35,6 +35,7 @@ export const SignedClaimType = t.type({
   signature: SignatureType,
 })
 
+export const domainTxtHeader = 'celo-site-verification'
```

Comment:
Optional Nit: Consider making the name all CAPS

Reply:
:+1: